portal-frontend/src/app/features/edit-submission/other-attachments/other-attachments.component.ts

```diff
@@ -52,9 +52,8 @@ export class OtherAttachmentsComponent extends FilesStepComponent implements OnI
       }
     });
 
-    this.loadDocumentCodes();
-
     this.$applicationDocuments.pipe(takeUntil(this.$destroy)).subscribe((documents) => {
+      this.loadDocumentCodes();
```

Comment:
the same with codes, since they are static, do you need to reload them every time you reload documents?